esp-hal/CHANGELOG.md

```diff
@@ -28,7 +28,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Make saving and restoring SHA digest state an explicit operation (#2049)
 - Reordered RX-TX pairs in all APIs to be consistent (#2074)
-- Make saving and restoring SHA digest state an explicit operation (#2049)
```

Comment:
Intentional?

Reply:
nope, and it won't stay, but there's no point in rebasing this PR right now